### PR TITLE
Add azure-db command

### PR DESCRIPTION
**Pull request by mw2q**
Deploy architectures on Azure Database for PostgreSQL.

Change the RDS PgType to DBaaS as many of its code paths can be shared
with Azure Database.

### CODE REVIEW

File: edbdeploy/data/terraform/azure-db/environments/vm/vm.tf [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  tags = var.project_tags
+}
+
+resource "azurerm_postgresql_server" "postgresql_server" {
+  name = format("%s-server", var.cluster_name)
+  location = var.azure_region
+  resource_group_name = var.resourcegroup_name
+ 
+  administrator_login          = "postgres"
+  administrator_login_password = "Ahb0uL!eerahmiT4ShooF6teepie2eu6"
```
Comment (jt-edb) [inline edbdeploy/data/terraform/azure-db/environments/vm/vm.tf:223]: There is absolutely no way to pass the password from the python code? Having hardcoded passwords makes me nervous.

File: edbdeploy/data/terraform/azure-db/environments/security/security.tf [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  security_rule {
+    name                       = "EDB-PgBouncer"
+    priority                   = 900
+    direction                  = "Inbound"
+    access                     = "Allow"
+    protocol                   = "Tcp"
+    source_port_range          = "*"
+    destination_port_range     = "6432"
+    source_address_prefix      = "*"
+    destination_address_prefix = "*"
```
Comment (jt-edb) [inline edbdeploy/data/terraform/azure-db/environments/security/security.tf:184]: Do we need all these security rules for `azure-db`?

**Review by jt-edb — COMMENTED** [review]
Overall LGTM, but still lot of duplicated and useless (I think) terraform code for barman, PEM, and pgpool.

File: edbdeploy/data/terraform/azure-db/environments/security/security.tf (hunk 2 above)
Comment (mw2q) [reply edbdeploy/data/terraform/azure-db/environments/security/security.tf:184]: We don't, will work on removing.

File: edbdeploy/data/terraform/azure-db/environments/vm/vm.tf (hunk 1 above)
Comment (mw2q) [reply edbdeploy/data/terraform/azure-db/environments/vm/vm.tf:223]: Oops, that was on my TODO.  Will work on it.

**Review by jt-edb — APPROVED** [review]

**Pull request merged**